src/App.js

```diff
@@ -19,6 +24,13 @@ class App extends Component {
         insights.chrome.auth.getUser().then(data => {
             this.setState({ identity: data.identity });
         });
+
+        api.getVersion().then(() => {
+            this.setState({ permission: true });
+        }).catch(error => {
+            this.setState({ permission: error.response.status !== 404 });
```

Comment:
I would default to being more restrictive and setting permission to `false` for all errors. (If I understand axios' documentation correctly, this handler is called for any non-2xx response.)

For example, the backend not being reachable or throwing internal server errors should also show the PermissionDenied message.